### ISSUE TITLE
`component` is not defined

### ISSUE THREAD

**Issue by stevenkuster** [break-stuff/wc-storybook-helpers#52]
Hi! Since version 1.3.0 the global `component` variable is broken in our Storybook. So it looks the fix that is mentioned in version 1.3.1 doesn't do the trick for us unfortunately.

**Comment by break-stuff** [comment]
Yeah, there was a timing issue there. I added a patch in 1.4.0. let me know if that is working for you.

**Comment by stevenkuster** [comment]
I see! That does seem to fix it mostly. I do however now run into the following problem. If I have something like this:
```
  render: (args: Args) => html`
    ${template(args)}
    <script>
      const checkboxes = component.querySelectorAll("checkbox-component");
    </script>
  `,
```
I get a `Uncaught SyntaxError: Identifier 'checkboxes' has already been declared`. Is that related?

And thanks for the very quick response btw. 🙌

**Comment by stevenkuster** [comment]
Hmm, I think I get why that happens. The story is visible twice on the page so that's probably why it errors. If I set a `type="module"` on the script tag the problem goes away.

**Comment by break-stuff** [comment]
Yep! Or you have to assign it to the `window` variable so it gets overridden for each story.

**Comment by stevenkuster** [comment]
Cool, let me close this issue. Thanks again!
